sci-visualization/mantid/files/mantid-3.3.0-minigzip-OF.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/MantidPlot/src/zlib123/minigzip.c b/MantidPlot/src/zlib123/minigzip.c
+index 97fbae1..5f4612f 100644
+--- a/MantidPlot/src/zlib123/minigzip.c
++++ b/MantidPlot/src/zlib123/minigzip.c
+@@ -50,6 +50,10 @@
+ #  include <unix.h> /* for fileno */
+ #endif
+ 
++#ifndef OF
++#  define OF(args) args
++#endif
++
+ #ifndef WIN32 /* unlink already in stdio.h for WIN32 */
+   extern int unlink OF((const char *));
+ #else
```

Comment:
That link doesn't work but I found a version in Google's cache. As far as I can tell all it wants me to do is write a few lines at the top of the file describing what the patch is doing and why.